allure-selenide/src/main/java/io/qameta/allure/selenide/AllureSelenide.java

```diff
@@ -60,40 +61,39 @@ public AllureSelenide savePageSource(final boolean savePageHtml) {
     }
 
     @Override
-    public void onEvent(final LogEvent event) {
-        lifecycle.getCurrentTestCase().ifPresent(uuid -> {
-            final String stepUUID = UUID.randomUUID().toString();
-            lifecycle.startStep(stepUUID, new StepResult()
-                    .setName(event.toString())
-                    .setStatus(Status.PASSED));
+    public void beforeEvent(final LogEvent event) {
+        stepUUID = UUID.randomUUID().toString();
```

Comment:
should it really set status to PASSED? Unlike in the old implementation, this is logged before event even happened.

Reply:
Previous logic not changed. Except `lifecycle.getCurrentTestCase` unwrap that was unused before.